shard/Dockerfile

```diff
@@ -41,18 +41,7 @@ RUN python3.8 -m pip install wheel
 RUN python3.8 -m pip install --trusted-host pypi.python.org -r requirements.txt
 #RUN apk del libpng-dev jpeg-dev libffi-dev libxml2-dev python3-dev libxslt-dev freetype-dev
 
-WORKDIR /pyre2
```

Comment:
uh
don't we still use this?

Reply:
or was that just a remnant from a previous era that's not doing anything

Reply:
I have no clue why that's there. All of pyre2 is set up and installed earlier in the Dockerfile.

Reply:
nice